src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -14,13 +15,13 @@ namespace MockBot.Api.Controllers
     public class ChatController : ControllerBase
     {
         private readonly IChatService _chatService;
-        private readonly IAsyncProcessorService<DmrRequest> _dmrService;
+        private readonly IAsyncProcessorService<DmrRequest> _processor;
         private readonly BotSettings _settings;
 
-        public ChatController(IChatService chatService, IAsyncProcessorService<DmrRequest> dmrService, BotSettings settings)
+        public ChatController(IChatService chatService, IAsyncProcessorService<DmrRequest> dmrProcessorService, BotSettings settings)
         {
             _chatService = chatService;
-            _dmrService = dmrService;
+            _processor = dmrProcessorService;
```

Comment:
Nittiest of Picks:  
_processor for both parameter and member variable?

Reply:
Valid point, are you okay with `dmrRequestProcessor` instead?